packages/ia-components/sandbox/bookreader-component/bookreader-wrapper-main.jsx

```diff
@@ -73,6 +81,7 @@ export default class BookReaderWrapper extends Component {
     } else {
       const br = new BookReader(fullOptions);
       window.br = br;
+      this.bookreader = window.br;
```

Comment:
QUESTION: Why is `br` effectively being stored in both `window.br` and `this.bookreader`?

Reply:
window.br is so globally used, we must keep it until we can truly remove the global dependency.